doc/support.rst

```diff
@@ -28,7 +28,7 @@ User questions
   tag.
 
 - For general theoretical or methodological Machine Learning questions
-  `metaoptimize.com/qa <http://metaoptimize.com/qa>`_ is probably a more
+  `stack exchange <http://meta.stackexchange.com/questions/130524/which-stack-exchange-website-for-machine-learning-and-computational-algorithms>`_ is probably a more
```

Comment:
sure, but I'm not sure that is actually more helpful for machine learning questions than the machine-learning tag on stackoverflow.